docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -0,0 +1,311 @@
+
+```bash
+databricks fs ls dbfs:/FileStore/iris-databricks/data
+```
+
+You should see the contents of the project's `data/` directory printed to your terminal:
+
+```bash
+01_raw
+02_intermediate
+03_primary
+04_feature
+05_model_input
+06_models
+07_model_output
+08_reporting
+```
+
+## Deploy and run your Kedro project using the workspace UI
+
+To run your packaged project on Databricks, login to your Databricks account and perform the following steps in the workspace:
+
+1. **Create a new job**: In the Databricks workspace, navigate to the `Worfklows` tab and click `Create Job`:
+
+![Create Databricks job](../../meta/images/databricks_create_new_job.png)
+
+2. **Create a new job cluster**: Create a dedicated [job cluster](https://docs.databricks.com/clusters/index.html) to run your job by clicking on the drop-down menu in the `Cluster` field:
+
+![Create Databricks job cluster](../../meta/images/databricks_create_job_cluster.png)
+
+Configure the job cluster with the following settings:
+
+- In the `name` field enter `kedro_deployment_demo`.
+- Select the radio button for `Single node`.
+- Select the runtime `12.2 LTS` in the `Databricks runtime version` field.
+- Leave all other settings with their default values in place.
+
+The final configuration for the job cluster should look the same as the following:
+
+![Configure Databricks job cluster](../../meta/images/databricks_configure_job_cluster.png)
+
+3. **Configure the job**: Configure the job with the following settings:
+
+- Enter `iris-databricks` in the `Name` field.
+- In the dropdown menu for the `Type` field, select `Python wheel`.
+- In the `Package name` field, enter `iris_databricks`. This is the name of your package as defined in your project's `src/setup.py` file.
+- In the `Entry Point` field, enter `databricks_run`. This is the name of the [entry point](#create-an-entry-point-for-databricks) to run your package from.
+- Ensure the job cluster you created in step two is selected in the dropdown menu for the `Cluster` field.
+- In the `Dependent libraries` field, click `Add` and upload [your project's `.whl` file](#package-your-project), making sure that the radio buttons for `Upload` and `Python Whl` are selected for the `Library Source` and `Library Type` fields.
+- In the `Parameters` field, enter the following list of runtime options:
+
+```bash
+["--conf-source", "/dbfs/FileStore/iris-databricks/conf", "--package-name", "iris_databricks"]
+```
+
+The final configuration for your job should look the same as the following:
+
+![Configure Databricks job](../../meta/images/databricks_configure_new_job.png)
+
+Click `Create` and then `Confirm and create` in the following pop-up asking you to name the job.
+
+4. **Run the job**: Click `Run now` in the top-right corner of your new job's page to start a run of the job. The status of your run can be viewed in the `Runs` tab of your job's page. Navigate to the `Runs` tab and track the progress of your run:
```

Comment:
At this point I got an error:

> `Run result unavailable: run failed with error message
 Unexpected failure while waiting for the cluster (0529-113546-pocghr6r) to be ready: Cluster 0529-113546-pocghr6r is in unexpected state Terminated: SECRET_RESOLUTION_ERROR(SERVICE_FAULT): databricks_error_message:Failed to fetch secrets referred to in Env Vars.`

<img width="1214" alt="image" src="https://github.com/kedro-org/kedro/assets/316517/201aadee-3921-4b41-9f5b-13bd130746ed">

Reply:
See my comment above for why this happened!

Really appreciate you testing these docs manually 🙂.